doc/api/synopsis.md

```diff
@@ -9,9 +9,62 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Commands displayed in this document are shown starting with `$` or `>` 
+to replicate how they would appear in a user's terminal. 
+Do not include the `$` and `>` character.
+They are there to indicate the start of each command. 
+
+There are many tutorials and examples that follow this 
+convention: `$` or `>` for commands run as a regular user, and `#` 
+for commands that should be executed as an administrator. 
+
+Lines that don’t start with `$` or `>` character are typically showing 
+the output of the previous command.
+
+Firstly, make sure to have downloaded Node.js from [Node.js Official website][].
```

Comment:
I personally do not like to recommend the regular download as it makes it more difficult for the user to switch to a new version. Therefore I would prefer to directly reference the guide.
It could be written as e.g.:
`Firstly, make sure to have downloaded and installed Node.js. See [this guide][] for further install information.`